src/com/untamedears/PrisonPearl/PrisonPearlCommands.java

```diff
@@ -604,6 +604,10 @@ private boolean imprisonCmd(CommandSender sender, String args[]) {
         	uuid = NameAPI.getUUID(args[0]);
         else
         	uuid = Bukkit.getOfflinePlayer(args[0]).getUniqueId();
+        if (uuid == null){
```

Comment:
This check isn't exactly foolproof.  For example Bukkit.getofflineplayer will always return an object regardless if the player has played or not.  It won't know if the player is real or not and the .getuniqueid will always return a value.